### ISSUE TITLE
[accessibility] files: cannot select multiple files with keyboard only

### ISSUE THREAD

**Issue by susnux** [nextcloud/server#62250]
When only using the keyboard it is currently not possible in the files app to select more than one file or folder.
The focus is locked (focus trap) to the files-list-actions.

https://github.com/user-attachments/assets/b958c18b-b419-4c47-8569-98020d7f45c8

cc @skjnldsv as I guess this is a regression from fixing the files list actions in https://github.com/nextcloud/server/pull/58457
